Manifest.toml

```diff
@@ -139,9 +139,11 @@ version = "0.4.15"
 
 [[JLLWrappers]]
 deps = ["Preferences"]
-git-tree-sha1 = "642a199af8b68253517b80bd3bfd17eb4e84df6e"
+git-tree-sha1 = "71a9713ecb703d4955d9facf4cfe26d2a4ee3f72"
+repo-rev = "vc/platform_tags"
+repo-url = "https://github.com/JuliaPackaging/JLLWrappers.jl.git"
```

Comment:
I think we should merge https://github.com/JuliaPackaging/JLLWrappers.jl/pull/35 before actually merging this PR, and use here a released version.